### PR TITLE
Support for Decidim 0.19.x

### PR DESCRIPTION
**Pull request by ahukkanen**
I've updated the module to support Decidim 0.19.x, fixed the issues and tests.

I've also branched current master to `0.18-stable` in order to use that for Decidim 0.18.x.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,16 +19,23 @@ jobs:
     steps:
       - checkout
 
+      - run:
+          name: Install bundler
+          command: |
+            gem update --system
+            gem uninstall bundler
+            gem install bundler --version $BUNDLER_VERSION
```
Comment (armandfardeau) [inline .circleci/config.yml:27]:
```suggestion
            export BUNDLER_VERSION=$(cat Gemfile.lock | tail -1 | tr -d " ")
            gem install bundler --version $BUNDLER_VERSION
```

**Review by armandfardeau — CHANGES_REQUESTED** [review]
Glad you worked on this! I'm not a big fan of the hardcoded bundler version. What do you think of my suggestion?

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,9 +5,10 @@ version: 2
 jobs:
   build:
     docker:
-      - image: decidim/decidim@sha256:348fb1faa49cfe76ba7cbeafc67ca09002cf95f0ffa7a76eefa9d5b3774e0ee9
+      - image: decidim/decidim@sha256:dcf729c93abf361ecedc0eadf4fc208a2a990dd5fe85e708437a6fba9456682a
         environment:
           DATABASE_USERNAME: postgres
+          BUNDLER_VERSION: 2.0.2
```
Comment (armandfardeau) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml (hunk 1 above)
Comment (ahukkanen) [reply .circleci/config.yml:27]: Fine by me!

**Review by armandfardeau — CHANGES_REQUESTED** [review]

**Pull request merged**